src/shared/style/_icons.scss

```diff
@@ -1,7 +1,7 @@
-.co-icon-space-l {
+.hacDev-icon-space-l {
```

Comment:
Wasn't able to find this class being used anywhere, would it be okay to remove it?

Reply:
Removed it.